boards/common/cc2538/Makefile.include

```diff
@@ -1,9 +1,6 @@
 # define the default flash-tool
 PROGRAMMER ?= cc2538-bsl
 
-# setup serial terminal
-include $(RIOTMAKE)/tools/serial.inc.mk
-
 # For backward compatibility
```

Comment:
I see you moved it but this would not work know. But IMO we can deprecate, it was done +3 releases ago.

Reply:
Ah no forget what I said, this is only if `PORT_BSL` is set